src/utilities/time.ts

```diff
@@ -66,3 +66,20 @@ export const getLocalTimeStamp = (time: number) => {
   const fullSeconds = (`0${seconds}`).slice(-2);
   return `${fullDay}${abbrMonth}${abbrYear}-${fullHours}:${fullMinutes}:${fullSeconds}`;
 };
+
+export const getDisplayTimeDate = (time: number) => { //returns 12:05 PM Nov 14
+  const start = new Date(time);
+  const month = start.getMonth();
+  const day = start.getDate();
+  const hours = start.getHours();
+  const minutes = start.getMinutes();
+  const fullMinutes = (`0${minutes}`).slice(-2);
+  const ampm = hours >= 12 ? 'PM' : 'AM';
+  const hours12 = hours > 12 ? hours - 12 : hours === 0 ? 12 : hours;
+  const full12Hours = (`0${hours12}`).slice(-2);
```

Comment:
```suggestion
  const full12Hours = (`${hours12}`).slice(-2);
```
I don't think you want the leading `0` on the hour.